app/Models/Lead.php

```
@@ -367,6 +368,80 @@ public function syncCategories(array $categoryIds): void
+    /**
+     * @return list<string>
+     */
+    public function resolvedPreferredContactTimes(): array
+    {
+        $stored = $this->preferred_contact_times;
+        if (is_array($stored) && $stored !== []) {
+            return PreferredContactTime::normalizeList($stored);
+        }
+
+        return PreferredContactTime::normalizeList(
+            $this->preferred_contact_time?->value ?? $this->preferred_contact_time
```

PHPStan (2.2.7)

[error] Cannot access property $value on string.


(property.nonObject)